integration-tests/main/src/main/resources/application.properties

```diff
@@ -66,3 +66,6 @@ tester-config-exp=tester
 
 # used to make sure that @QuarkusIntegrationTest works properly when basic logging is turned off
 %prod.quarkus.log.category."io.quarkus".level=WARN
+
+#Necessary for assertions in UsedCodeReportITCase:
+quarkus.native.enable-reports=true
```

Comment:
I have no problems with us adding the new property, but I didn't understand why was it useful to introduce it?

Reply:
Without this `native-image` doesn't generate reports, and without the reports we can't count the used classes/methods/packages.

An alternative would be to use the `native-image` output (which [we also dump as a json file](https://github.com/quarkusio/quarkus/blob/c6782ff028338f037b10ad1fce4340dc01f3208a/core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java#L796) with) instead of the reports. I guess that would be better. I will work on it if you agree, but that will make sense only if we are OK with https://github.com/quarkusio/quarkus/pull/36091#issuecomment-1731111572

Reply:
Yes, please use the regular build output (which should be anabled by default) and look at the json to do the assertions. That's what those metrics are meant for.